roles/vagrant_kubernetes/tasks/main.yml

```diff
@@ -1,3 +1,21 @@
+- name: set iptable rules for nfs
+  iptables:
+    chain: INPUT
+    in_interface: "{{ ansible_default_ipv4.interface }}"
```

Comment:
I'd try

    in_interface: "!{{ ansible_default_ipv4.interface }}"

to say all interfaces except default one.